ppu/ppu.go

```diff
@@ -9,9 +15,12 @@ type PPU struct {
 	colorMath      *colorMath      // Color math parameters
 	m7             *m7             // mode 7 parameters
 	display        *display
+	window         [2]*window
+	status         *status
+	registers      [0x40]register // registers represents the ppu registers as methods
 
-	registers [0x40]register // registers represents the ppu registers as methods
-	window    [2]*window
+	hCounter uint16
```

Comment:
Why not move hCounter and vCounter to the status struct directly ? ( It's not related ?)

Reply:
Hcounter and hcounter will be used later in the code to generate lines so their more general. I'm not sure what to do with them though